src/Microsoft.ML.Transforms/Text/WordBagTransform.cs

```diff
@@ -94,7 +94,7 @@ internal bool TryUnparse(StringBuilder sb)
         /// </summary>
         public sealed class TokenizeColumn : OneToOneColumn { }
```

Comment:
>public [](start = 8, length = 6)

also internal

Reply:
The class is internal, so it should be fine

---
In reply to: [254030739](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254030739) [](ancestors = 254030739)

Reply:
deleted actually. had 0 references :)

---
In reply to: [254030739](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254030739) [](ancestors = 254030739)